Computing editableValues

andresfulla opened the pull request:
@darquesdev

xbrianlee committed:
LRQA-52608 Make macrodef more specific

AliciaGarciaGarcia committed:
LPS-103067 Test get page title, sufix and full page title

xbrianlee committed:
LRQA-52608 Configure test properties for Kibana

xbrianlee committed:
LRQA-52608 Move configure-kibana-properties as it's dependent on X-Pack

ling-alan-huang committed:
LPS-103872 Delimit valuse using hard line breaks

xbrianlee committed:
LRQA-52608 Pass in index name & Elasticsearch port

ling-alan-huang committed:
LPS-103872 Fix regex

Commit:
LRQA-52909 Make macrodef more specific on ES6

Commit:
LRQA-52992 Add learning to rank smoke test on ES 7

Commit:
LRQA-52990 Add learning to rank smoke test on ES 6

ling-alan-huang committed:
LPS-103872 Skip normal sentence

ling-alan-huang committed:
LPS-103872 Fix regex

Commit:
LRQA-52909 Add elasticsearch6 ltr test properties

Commit:
LRQA-52909 Add Elasticsearch 6.7.1 test properties

Commit:
LRQA-52909 Add macrodef for setting up ES LTR on ES6

Commit:
LRQA-52909 Move configure-kibana-properties as it's dependent on X-Pack

Commit:
LRQA-52909 Add target for restoring Elasticsearch backups

Commit:
LRQA-52909 Unzip LTR linear regression model

Commit:
LRQA-52608 Add LTR properties

Commit:
LRQA-52608 Add macrodef for setting up Elasticsearch LTR

Commit:
LRQA-52608 Add target for restoring Elasticsearch backups

Commit:
LRQA-52608 Unzip LTR linear regression model

Commit:
LRQA-52909 Remove inputstring parameter

Commit:
LRQA-52909 Move configure-kibana-properties below

Commit:
LRQA-52909 Update elasticsearch6 properties

darquesdev committed:
LPS-102884 Just taking into account the first experience. If it does not exist 'defaultValue' is returned

darquesdev committed:
LPS-102884 Do not copy defaultValue since this is already available within the editableValue

darquesdev committed:
LPS-102884 Fix test

xbrianlee committed:
LRQA-52608 Disable xpack.ml otherwise ES startup may fail

xbrianlee committed:
LRQA-52608 Add additional permissions to elasticsearch.dir

xbrianlee committed:
LRQA-52608 Add input string for plugin installation prompt

xbrianlee committed:
LRQA-52992 Auto SF

ling-alan-huang committed:
LPS-103872 SF

ling-alan-huang committed:
LPS-103872 Fix regex, only match the filename, don't match the path that contains "portal-", e.g. "portal-impl/src/system.propertie"

xbrianlee committed:
LRQA-52909 Unset all properties that we set

Commit:
LPS-102884 Methods to retrieve segmented content

hhuijser committed:
LPS-103466 Move method getImportNames to BaseJavaTermCheck

hhuijser committed:
LPS-103466 Methods with annotation Activate/Deactivate should be protected

hhuijser committed:
LPS-103466 Enforce method names for activate/deactivate

hhuijser committed:
LPS-103466 SF

hhuijser committed:
LPS-103466 Add method getLineNumber(int pos)

hhuijser committed:
LPS-103466 Apply

ling-alan-huang committed:
LPS-103872 Skip logic to put on multiple lines when property name contains "regex" instead of putting in source-formatter.properties

ling-alan-huang committed:
LPS-103872 Skip adding empty line if the property key in next line is same with current property key

xbrianlee committed:
LRQA-52992 Disable bootstrap.system_call_filter due to SecComp failure on CentOS

xbrianlee committed:
LRQA-52992 Add permissions for Elasticsearch directory in portal

darquesdev committed:
LPS-102884 Refactor extracting methods to facilitate testing

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

AliciaGarciaGarcia committed:
LPS-103067 Test get canonical layout URL

Commit:
LRQA-52992 Update the directory

ling-alan-huang committed:
LPS-103872 Fix regex, use possessive quantifier

ling-alan-huang committed:
LPS-103872 Auto SF

Commit:
LRQA-52608 Disable xpack.ml on elasticsearch6

Commit:
LRQA-53292 Ignore ci test until LRQA tickets resolved

hhuijser committed:
LPS-103466 Auto SF

hhuijser committed:
LPS-103466 Sort

hhuijser committed:
LPS-103466 Add suppressions. These are breaking tests

hhuijser committed:
LPS-103252 No need to specify types when both key/value type are final

hhuijser committed:
LPS-103252 SF

hhuijser committed:
LPS-103252 Apply to LinkedHashMap

hhuijser committed:
LPS-103252 Apply to LinkedHashMap

xbrianlee committed:
LRQA-52992 Typo

hhuijser committed:
LPS-103252 Simplify

hhuijser committed:
LPS-103252 Add generics

hhuijser committed:
LPS-103252 Few more

hhuijser committed:
LPS-103252 Add more put calls to the chain

hhuijser committed:
LPS-103252 No need to declare variable if it is returned right after

hhuijser committed:
LPS-103252 Apply to TreeMap

hhuijser committed:
LPS-103872 SF

brookedalton committed:
LRQA-53184 Modify find a page paths

mwilliams2014 committed:
LPS-95820 update bundle.properties

arboliveira committed:
LPS-98533 portal-search-similar-results-web: Fix: Unprocessed exception

https://issues.liferay.com/browse/LPS-98533
https://issues.liferay.com/browse/LPS-103985

mtambara committed:
LPS-104004 Add service references to ensure web will register service resource permissions

brookedalton committed:
LRQA-53182 Add macro that goes to page template in product navigation menu

brookedalton committed:
LRQA-53182 Update testcases to go to new page template location

sandrodw3 committed:
LPS-103884 Removed glass icon from SearchForm component

austinchiang committed:
LRQA-53331 Navigate to admin

austinchiang committed:
LRQA-53331 Skip portal instance setup

brianwulbern committed:
LRQA-53330 Update 7.x poshi runner jar to latest

austinchiang committed:
LRQA-53331 Use conditional to accommodate for DM and WC

liulukiwi committed:
LRQA-53292 Remove ignore for head past and add ignore for one new LRQA

austinchiang committed:
LRQA-53331 Account for if the product menu is closed

Commit:
LRQA-52832 Regenerate class-site-template-7.2.lar folder

liuxingye committed:
LRQA-53344 Add the image before publishing

jkappler committed:
LPS-103987 Fix action command name

Hanlf committed:
LRQA-53287 Remove unnecessary properties

AliciaGarciaGarcia committed:
LPS-100969 Change SEO configuration Open Graph messages

AliciaGarciaGarcia committed:
LPS-100969 BL

wincent committed:
LPS-96723 Remove pointless comma operator

This pattern:

    fn(...(null, args));

looks like a mistranslation of:

    fn.apply(null, args);

ie. call function `fn` with its context (`this`) set to `null`, and
passing it `args`.

However, the comma operator has no effect. It evaluates the first
operand (`null`), discards it, then evaluates the second one (`args`)
and uses that as the result of the expression:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator

In other words, it can be rewritten as just:

    fn(...args);

adolfopa committed:
LPS-103875 Add default resources

adolfopa committed:
LPS-104066 Update dependency

jorgediaz-lr committed:
LPS-104076 Avoid NPE in case there is no DL processor available

AliciaGarciaGarcia committed:
LPS-104070 Fix grammar in Open Graph tooltips

AliciaGarciaGarcia committed:
LPS-104070 BL

jorgediaz-lr committed:
LPS-104076 SF

andresfulla commented:
@p2kmgcl can you take a look to the last commit a please?

andresfulla commented on modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/content/MapContentForm.es.js:
```diff
@@ -22,6 +22,7 @@ import {
 	EDITABLE_FRAGMENT_ENTRY_PROCESSOR,
 	FRAGMENTS_EDITOR_ITEM_TYPES
 } from '../../utils/constants';
+import {computeEditableValue} from '../../utils/segmentedValues.es';
```
`segmentedValues` is not the name of the year

darquesdev committed:
LPS-102884 Add tests

andresfulla commented on modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/utils/segmentedValues.es.js:
```diff
@@ -0,0 +1,107 @@
+	return result;
+}
+
+export function computeConfigurationEditableValue(
```
- [ ] Docu

p2kmgcl commented on modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js:
```diff
@@ -54,7 +54,7 @@ import {
 } from '../../utils/constants';
 import {isDropZone} from '../../utils/isDropZone.es';
-import {prefixSegmentsExperienceId} from '../../utils/prefixSegmentsExperienceId.es';
+import {computeConfigurationEditableValuementedConfigurationValues} from '../../utils/segmentedValues.es';
```
what?

andresfulla replied:
that might not work as intended

p2kmgcl commented on modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js:
```diff
@@ -161,10 +154,10 @@ class FragmentEntryLink extends Component {
-					configurationValues[segmentsExperienceId] ||
-					configurationValues[defaultSegmentsExperienceId] ||
-					configurationValues;
+				const segmentedConfigurationValues = computeConfigurationEditableValuementedConfigurationValues(
```
![image](https://user-images.githubusercontent.com/800645/68119034-ded40d80-ff01-11e9-8598-ea0c2e2c742f.png)

pavel-savinov committed:
LPS-103602 Don't show pages tree icon in case when user doesn't have appropriate permission

adolfopa committed:
LPS-103322 No need to escape

This is either printed in contexts where escaping is not necessary
(e.g. the browser bar), or escaped on usage.

liferay-translation committed:
LPS-77699 Update Translations

JoyceWang08211 committed:
LRQA-51432 Add case to subscribe web content folder under staging

xbrianlee committed:
LRQA-52992 Ignore LTR test until LPS-104072 is fixed

adolfopa committed:
LPS-103956 Add finder method

adolfopa committed:
LPS-103956 Autogenerated

adolfopa committed:
LPS-103956 Delete entries by groupId

adolfopa committed:
LPS-103956 Add Override annotations

adolfopa committed:
LPS-103956 Remove rels before deleting a site

adolfopa committed:
LPS-103956 Write failing test

brianchandotcom committed:
LPS-103067 sort

kyle-miho committed:
LPS-104029 Rename usages of DLFileEntryInfoDisplayContributor to FileEntryInfoDisplayContributor

petershin committed:
LPS-103170 Set compile.uptodate property

brianchandotcom committed:
LPS-103956 more accurat

brianchandotcom committed:
LPS-103956 regen

brianchandotcom committed:
LPS-103956 Good enough

kyle-miho committed:
LPS-104029 Use FileEntry class name instead of DLFileEntry

xbrianlee committed:
LRQA-52992 Sort

petershin committed:
LPS-104121

joshchong committed:
LRQA-53377 Account for upstream acceptance being a suite

joshchong committed:
LRQA-53377 Run ES6 by default if search engine not specified

brianchandotcom committed:
LPS-104121 prep next

brianchandotcom committed:
LPS-103872 prep next

brianchandotcom committed:
LPS-103872 prep next

brianchandotcom committed:
Revert "LPS-104066 Update dependency"

This reverts commit 333739c58cfac510a69c76c6a2cb9d9e0f0aa0e4.
